imports/api/playlists/server/publications.js

```diff
@@ -1,27 +1,32 @@
 import { Meteor } from 'meteor/meteor';
 import { Playlists } from '../playlists_collection.js';
-import { HTTP } from 'meteor/http';
-import { Random } from 'meteor/random';
 import { NowPlaying } from '../now_playing.js';
+import { pagination } from 'meteor/kurounin:pagination';
 
 Meteor.publish('playlist', function (id) {
   check(id, Number);
   return Playlists.find({ spinPlaylistId: id });
 });
 
-Meteor.publish('playlists', function(options) {
+Meteor.publish('playlists0', function(options) {
```

Comment:
Could you check to see if we are even using this publication in any subscription? If not, delete it. Otherwise, please rename it to `allPlaylists` or similar. Likewise for any other `whatever0` publications.